python/lib/communication/dmod/test/test_scheduler_client.py

```diff
@@ -88,15 +88,19 @@ def disable_logging(cls, level: Optional[int] = logging.ERROR):
         logging.disable(level)
 
     def setUp(self) -> None:
-        self.loop = asyncio.get_event_loop()
+        try:
```

Comment:
If we are using python 3.7 or newer, we can use [get_running_loop](https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.get_running_loop) here.

Also, should probably except just the `RuntimeError`?

Reply:
@hellkite500, yeah, that seems pretty sensible and easy to do.  I've push a change.